src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -418,6 +418,8 @@ uct_ud_verbs_ep_create_connected(uct_iface_h iface_h, const uct_device_addr_t *d
     ucs_status_t status, status_ah;
     struct ibv_ah_attr ah_attr;
 
+    UCS_OBJ_SIZE_LIMIT(uct_ud_verbs_ep_t, 320, 368);
```

Comment:
do we really need those checks in the lib, since we already have them in gtest?

Reply:
early error detection...
but ok, I will remove it from sources